frontends/concrete-python/examples/pir/pir_utils.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+from concrete import fhe
+
+
+@fhe.compiler({"one_hot_vector": "encrypted", "database": "clear"})
+def get_ith_element_of_database(one_hot_vector: np.ndarray, database: np.ndarray) -> int:
+    return np.dot(one_hot_vector, database)
+
+
+def make_one_hot_vector(index: int, size: int) -> np.ndarray:
+
+    answer = np.zeros(shape=(size,), dtype=np.int8)
+    answer[index] = 1
+    return answer
+
+
+def compile_function(database, show_mlir=False, show_graph=False):
+
+    get_ith_element_of_database.reset()
+
+    database_length = database.shape[0]
+    inputset_length = 100
```

Comment:
Could we simplify that by computing the worst case query?

Reply:
what does that mean